src/lib/flow/sol-flow-builder.h

```diff
@@ -60,6 +60,10 @@ int sol_flow_builder_del(struct sol_flow_builder *builder);
  * fallback to using the default resolver. */
 void sol_flow_builder_set_resolver(struct sol_flow_builder *builder, const struct sol_flow_resolver *resolver);
 
+/* Set type description to use. Input/output ports,
+ * and options descriptions are automatically set. */
+void sol_flow_builder_set_type_description(struct sol_flow_builder *builder, const char *name, const char *category, const char *description, const char *author, const char *url, const char *license, const char *version);
```

Comment:
document we keep references to these strings during the lifetime of the builder, so the user knows he must keep the strings alive or it will segv.